sig-node/sig-node-contributor-ladder.md

```diff
@@ -0,0 +1,116 @@
+
+### Reviewer
+Reviewer status is an indication that the person is committed to the SIG activities, technically sound, has accumulated enough context, and is overall trustworthy to help inform approvers and aid contributors. The requirements listed in the [membership document](../community-membership.md#reviewer) highlight this as well.
+
+The following is the breakdown of requirements listed in the membership document broken down to these categories:
+
+- Committed - proof of sustained contributions
+- member for at least 3 months
+- Technically sound
+- Primary reviewer for at least 5 PRs to the codebase
+- Reviewed or merged at least 20 substantial PRs to the codebase
+- Has enough context
+- Knowledgeable about the codebase
+- Trustworthy - established trust with the community
+- Sponsored by a subproject approver
+- With no objections from other approvers
+
+SIG Node establishes the following clarifications and additional requirements.
+
+**Committed**
+
+The 3 month of activities should be established looking at PRs reviews history.
+Active participating in SIG Node or SIG Node CI weekly meetings or other transient meetings that arise when tackling specific problems in future (exceptions are allowed for cases when timezone or other personal limitations are not allowing for the meeting participation)
+
+SIG Node recognizes that 3 months may not be enough to establish a deep understanding of the codebase. Reviewer status for a subfolder is a step towards SIG-wide reviewer. See the areas section of this document below.
+
+**Technically sound**
+
+Proof of primary reviewership and significant contributions must be provided. Nominees must provide the list of PRs (at least 5 for primary reviewer and 20 substantial PRs authored or reviewed) as suggested in the membership document. Here are additional comments for this list of PRs:
+
+- Reviewed PRs must be merged.
+- Since the purpose is to demonstrate the nominee's technical depth, PRs like analyzer warnings fixes, mechanical “find/replace”-type PRs, minor improvements of kubelet logging and insignificant bug fixes are valued, but not counted towards the reviewer status nomination. Lack of reviews of those PRs may be a red flag for nomination approval.
+- Making the most of reviews in a single area is a good indication that a subdirectory reviewer role may be preferred or top-level node reviewer.  Finding the right balance helps balance the nominees future workload.
+- A primary reviewer should drive the review of the PR without significant input / guidance from the approver or other reviewers. 
+
+**Has enough context**
```

Comment:
As a reader, it feels like having context blends in with the technical depth/being "technically sound" above?

Reply:
Agree, combined the sections.